model/ftn/wmesmfmd.ftn

```diff
@@ -6551,6 +6986,191 @@
 !/
       end subroutine CalcPStokes
 !/ ------------------------------------------------------------------- /
+#undef METHOD
+#define METHOD "ReadFromFile"
+      subroutine ReadFromFile (idfld, fldwx, fldwy, time0, timen, rc)
+!/
+!/                  +-----------------------------------+
+!/                  | WAVEWATCH III           NOAA/NCEP |
+!/                  |           U. Turuncoglu           |
+!/                  |                        FORTRAN 90 |
+!/                  | Last update :         18-May-2021 |
+!/                  +-----------------------------------+
+!/
+!/    DD-MMM-YYYY : Origination.                        ( version 7.XX )
```

Comment:
you can go ahead and fill in DD-MM-YYYY with today or other date (maybe the 18-May 2021)?

Reply:
do you mean date for Origination?